weave/trace_server/migrations/014_add_call_id_to_feedback.up.sql

```diff
@@ -0,0 +1,3 @@
+-- Add column and backfill weave_ref_id for all rows that have a weave_ref, the id is everything after the last '/'
+-- example: weave-trace-internal:///UHJvamVjdEludGVybmFsSWQ6MTA0MzQwOA==/call/0194f9bc-3347-7920-970e-580ac9ad21ed --> 0194f9bc-3347-7920-970e-580ac9ad21ed 
+ALTER TABLE feedback ADD COLUMN weave_ref_id String MATERIALIZED coalesce(regexpExtract(weave_ref, '/([^/]+)$'), '');
```

Comment:
We should note that this removes an implied, but not explicit property that you can leave feedback on properties of objects/calls. For example: comment on a row of a dataset. While I think this is probably OK, we should acknowledge this limitation.

Reply:
We can massage the regex to try to work in as many cases as possible, do you happen to know what those refs look like?